src/lib/setupContextUtils.js

```diff
@@ -1054,7 +1059,7 @@ function registerPlugins(plugins, context) {
         values: Object.keys(options.values ?? {}),
         hasDash: name !== '@',
         selectors({ modifier, value } = {}) {
-          let candidate = '__TAILWIND_PLACEHOLDER__'
+          let candidate = `TAILWINDPLACEHOLDER${id}`
```

Comment:
I don't love the placeholder, but I didn't love it before either haha. One potential improvement could be to use a UUID (without the `-`) but that's almost overkill for this.

```js
crypto.randomUUID().replaceAll('-','')
```

Reply:
yeah I really don't like the placeholder either but it's necessary for the current implementation. UUID would definitely be overkill for this.